internal/certification/certificates.go

```diff
@@ -30,7 +33,12 @@ const (
 )
 
 type Certificates struct {
-	Certificates map[string]map[string][]byte
+	// {
+	// 		ClientCertificateSecretKey: {
+	// 			"tls.crt": []byte
+	// 		}
+	// }
```

Comment:
nit: Is this comment describing usage? Or dead code...?

Reply:
Ah sorry will remove.  I think a little note on shape would be good here but I"ll reformat it to look less janky.